Automattic/wp-calypso
# Site Transfers: Make site transfer support link inline

Related to https://github.com/Automattic/dotcom-forge/issues/2553

## Proposed Changes

In this PR, I propose to make the support link inline to avoid users navigating from the site.

<img width="762" alt="Screenshot 2023-05-31 at 07 52 47" src="https://github.com/Automattic/wp-calypso/assets/727413/03b57df9-bf7b-4d9f-925f-af78119aee6e">


## Testing Instructions



1. Create a simple site
2. Navigate to General Settings
3. Click 'Transfer your site'
4. Click 'Learn More'
5. Confirm it opens in the popup which can be closed

## Pre-merge Checklist
